<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>seit</class>
 <widget class="QWidget" name="seit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>805</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*:hover{
color: rgb(0, 170, 255);
}
#widget{
border-radius:15px;
background-color: rgb(0, 0, 0);
}
#widget_show
{
background-color: rgba(0,0,0,128); 
}
*{
font: 100 12pt &quot;华文仿宋&quot;;
color: rgb(255, 255, 255);
border:0px;
background-color: transparent;
}
QComboBox QAbstractItemView 
{
background-color:rgba(0,0,0,128);
}
QMenu
{
background-color: rgba(0,0,0,128);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QPixmapShow" name="pixmapshow" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_show" native="true">
           <property name="minimumSize">
            <size>
             <width>361</width>
             <height>600</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QScrollArea" name="scrollArea">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>472</width>
                 <height>769</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QWidget" name="widget_source" native="true">
                  <layout class="QGridLayout" name="gridLayout">
                   <property name="bottomMargin">
                    <number>20</number>
                   </property>
                   <property name="horizontalSpacing">
                    <number>10</number>
                   </property>
                   <property name="verticalSpacing">
                    <number>20</number>
                   </property>
                   <item row="11" column="3" colspan="2">
                    <widget class="QWidget" name="widget_2" native="true">
                     <layout class="QHBoxLayout" name="horizontalLayout">
                      <item>
                       <widget class="QPushButton" name="pushButton_sizeUp">
                        <property name="styleSheet">
                         <string notr="true">image: url(:/move.png);</string>
                        </property>
                        <property name="text">
                         <string/>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="pushButton_sizeDown">
                        <property name="styleSheet">
                         <string notr="true">image: url(:/min.png);</string>
                        </property>
                        <property name="text">
                         <string/>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item row="7" column="0">
                    <widget class="QLabel" name="label_3">
                     <property name="text">
                      <string>桌面歌词</string>
                     </property>
                    </widget>
                   </item>
                   <item row="3" column="3">
                    <widget class="QPushButton" name="pushButton_lrc_sizeAdd">
                     <property name="styleSheet">
                      <string notr="true">image: url(:/move.png);</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="0">
                    <widget class="QLabel" name="label">
                     <property name="text">
                      <string>音乐存放位置</string>
                     </property>
                    </widget>
                   </item>
                   <item row="3" column="1" colspan="2">
                    <widget class="QLabel" name="label_8">
                     <property name="text">
                      <string>滚动歌词字体大小</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="1" colspan="3">
                    <widget class="QPushButton" name="pushButton_delete_dir">
                     <property name="text">
                      <string>删除地址</string>
                     </property>
                    </widget>
                   </item>
                   <item row="12" column="1" colspan="2">
                    <widget class="QCheckBox" name="checkBox_downLrc">
                     <property name="text">
                      <string>下载歌词（网易云）</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1" colspan="4">
                    <widget class="QComboBox" name="comboBox_all_url"/>
                   </item>
                   <item row="11" column="2">
                    <widget class="QLabel" name="label_lrcShow_fontSize">
                     <property name="text">
                      <string>20</string>
                     </property>
                    </widget>
                   </item>
                   <item row="7" column="3" colspan="2">
                    <widget class="QPushButton" name="pushButton_downLrc">
                     <property name="text">
                      <string>下一句歌词颜色</string>
                     </property>
                    </widget>
                   </item>
                   <item row="9" column="1" colspan="4">
                    <widget class="QFontComboBox" name="fontComboBox_lrc"/>
                   </item>
                   <item row="11" column="1">
                    <widget class="QLabel" name="label_2">
                     <property name="text">
                      <string>字体大小：</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="4">
                    <widget class="QPushButton" name="pushButton_add_dir">
                     <property name="text">
                      <string>新加地址</string>
                     </property>
                    </widget>
                   </item>
                   <item row="7" column="1" colspan="2">
                    <widget class="QPushButton" name="pushButton_nowLrc">
                     <property name="text">
                      <string>当前歌词颜色</string>
                     </property>
                    </widget>
                   </item>
                   <item row="3" column="0">
                    <widget class="QLabel" name="label_4">
                     <property name="text">
                      <string>主界面设置</string>
                     </property>
                    </widget>
                   </item>
                   <item row="3" column="4">
                    <widget class="QPushButton" name="pushButton_lrc_sizeDecrease">
                     <property name="styleSheet">
                      <string notr="true">image: url(:/min.png);</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item row="12" column="0">
                    <widget class="QLabel" name="label_6">
                     <property name="text">
                      <string>在线服务</string>
                     </property>
                    </widget>
                   </item>
                   <item row="12" column="3" colspan="2">
                    <widget class="QCheckBox" name="checkBox_downCover">
                     <property name="text">
                      <string>下载封面（网易云）</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPixmapShow</class>
   <extends>QWidget</extends>
   <header location="global">qpixmapshow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
